requirements_dev.txt

```diff
@@ -1,4 +1,4 @@
-Django~=2.0
+Django~=3.0
```

Comment:
Please allow this to remain the lowest possible version for ensuring the backwards-compatibility issue mentioned above, meaning `2.2`.